### DISCUSSION TITLE
Ignoring 't' when iteration over channels_db?

### DISCUSSION THREAD

**Discussion by pebel76260** [danielhrisca/asammdf#574] [Ideas]
Hello everyone, 
to extract a list of all cgis, I iterate over channels_db. (for signalName, gi in mdf_obj.channels_db.items())
As 't' is no data channel, I have to ignore it when assembling my data dict. Wouldn't it be better if 't' was not listed here?

**Comment by danielhrisca** [comment]
All the channels must be appear in the ``channels_db``.

As an alternative did you try using the [iter_channels](https://asammdf.readthedocs.io/en/latest/api.html#asammdf.mdf.MDF.iter_channels) method?
